core/src/main/resources/defaults/blueprints/cdp-data-engineering-ha-702.bp

```diff
@@ -0,0 +1,403 @@
+{
+  "description": "7.0.2 - Data Engineering HA",
+  "blueprint": {
+    "cdhVersion": "7.0.2",
+    "displayName": "dataengineering ha",
+    "services": [
+      {
+        "refName": "zookeeper",
+        "serviceType": "ZOOKEEPER",
+        "roleConfigGroups": [
+          {
+            "refName": "zookeeper-SERVER-BASE",
+            "roleType": "SERVER",
+            "base": true
+          }
+        ]
+      },
+      {
+        "refName": "hdfs",
+        "serviceType": "HDFS",
+        "serviceConfigs": [
+          {
+            "name": "redaction_policy_enabled",
+            "value": "false"
+          },
+          {
+            "name": "zookeeper_service",
+            "ref": "zookeeper"
+          }
+        ],
+        "roleConfigGroups": [
+          {
+            "refName": "hdfs-NAMENODE-BASE",
+            "roleType": "NAMENODE",
+            "base": true
+          },
+          {
+            "refName": "hdfs-FAILOVERCONTROLLER-BASE",
+            "roleType": "FAILOVERCONTROLLER",
+            "base": true
+          },
+          {
+            "refName": "hdfs-JOURNALNODE-BASE",
+            "roleType": "JOURNALNODE",
+            "base": true
+          },
+          {
+            "refName": "hdfs-DATANODE-BASE",
+            "roleType": "DATANODE",
+            "base": true
+          },
+          {
+            "refName": "hdfs-SECONDARYNAMENODE-BASE",
+            "roleType": "SECONDARYNAMENODE",
+            "configs": [
+              {
+                "name": "fs_checkpoint_dir_list",
+                "value": "/should_not_be_required_in_HA_setup"
```

Comment:
Not required because?, and if so - this is probably not what the value needs to be?

Reply:
There is an open jira about this https://jira.cloudera.com/browse/OPSAPS-50592

Reply:
This setting only applies to the secondary namenode, which is not running in the HDFS HA setup.  It is only included in the template because CM requires it anyway.

Reply:
Thanks @adoroszlai